environments/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "test",
+  "description": "Delivery Cluster Environment",
+  "json_class": "Chef::Environment",
+  "chef_type": "environment",
+  "override_attributes": {
+    "delivery-cluster": {
+      "accept_license": true,
+      "id": "test",
+      "driver": "openstack",
+        "fog:OpenStack": {
```

Comment:
however because the actual chef provisioning DRIVER url includes multiple colons, we have to add 'fog:OpenStack' as the lookup for the attributes